features/flows/api/src/main/java/org/opennms/netmgt/flows/api/Converter.java

```diff
@@ -30,6 +30,8 @@
 
 import java.util.List;
 
+// TODO: Remove the converter once proto maps to Flow interface directly.
 public interface Converter<P> {
     List<Flow> convert(P packet);
+    Flow convertTo(P packet);
```

Comment:
Not sure if this will work out well. In sflow it's still possible to return multiple flows from one incoming message.
`Collections.singletonList` should make it easy to implement the original interface.